addon/bootstrap.js

```diff
@@ -11,46 +11,25 @@ XPCOMUtils.defineLazyModuleGetter(
   "resource://gre/modules/Services.jsm",
 );
 
-const STUDY = "button-icon-preference";
+const STUDY = "search-nudges";
 
 XPCOMUtils.defineLazyModuleGetter(
   this,
   "config",
-  `resource://${STUDY}/Config.jsm`,
+  `chrome://${STUDY}/content/Config.jsm`,
 );
 XPCOMUtils.defineLazyModuleGetter(
   this,
   "studyUtils",
-  `resource://${STUDY}/StudyUtils.jsm`,
+  `chrome://${STUDY}/content/StudyUtils.jsm`,
 );
 XPCOMUtils.defineLazyModuleGetter(
   this,
   "Feature",
-  `resource://${STUDY}/lib/Feature.jsm`,
+  `chrome://${STUDY}/content/lib/Feature.jsm`,
```

Comment:
The `content` directive is still supported for bootstrap addons, so I simply make use of it.